src/controllers/action-queue/cleaners.ts

```diff
@@ -63,13 +64,16 @@ const asError = asCodec(
   asJsonError
 )
 
+const asActionOpDisplayKey = asValue('create', 'swap-deposit', 'swap-deposit-fees')
+
 const asSeqActionOp = asObject<SeqActionOp>({
   type: asValue('seq'),
   actions: asArray((raw: any) => asActionOp(raw))
 })
 const asParActionOp = asObject<ParActionOp>({
   type: asValue('par'),
-  actions: asArray((raw: any) => asActionOp(raw))
+  actions: asArray((raw: any) => asActionOp(raw)),
+  displayKey: asOptional(asActionOpDisplayKey)
```

Comment:
Just a trick to use for if we were to have distinct `*ActionOpDisplayKey` types (my final ask in this review):

```ts
const asParActionOp = asObject<ParActionOp>({
  type: asValue('par'),
  actions: asArray((raw: any) => asActionOp(raw)),
  displayKey: (raw: unknown) => asOptional(asParActionOpDisplayKey)(raw)
})
const asParActionOpDisplayKey = asValue('create')
```

This extra anonymous arrow-function is how we can reference a cleaner before it's defined.